src/Response/DTO/Address.php

```diff
@@ -6,68 +6,29 @@
 
 class Address
```

Comment:
maybe you can make these classes readonly

Reply:
Sure, I fixed it.